pages/docker_compose.md

```diff
@@ -61,6 +61,16 @@ If you are using Docker Machine on Mac or Windows, your Docker daemon has only l
 
 </div>
 
+
```

Comment:
I think it would also happen for people that want to install a JHipster module in docker.